sound/soc/sof/intel/hda-stream.c

```diff
@@ -643,14 +644,6 @@ int hda_dsp_stream_init(struct snd_sof_dev *sdev)
 				SOF_HDA_SPIB_MAXFIFO;
 		}
 
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_DRSM)
-		/* FIXME: Remove? HDAC doesn't use DRSM so has no drsm_addr */
-		/* do we support DRSM */
-		if (sdev->bar[HDA_DSP_DRSM_BAR])
-			stream->drsm_addr = sdev->bar[HDA_DSP_DRSM_BAR] +
-				SOF_HDA_DRSM_BASE + SOF_HDA_DRSM_INTERVAL * i;
-#endif
-
 		hstream = &stream->hstream;
```

Comment:
What was this about? I don't even know what DRSM is, maybe worth sharing and educating the rest of us?

Reply:
That is for HDA DMA resume capabilities, which support resume from DPIB/LPIB, we haven't enabled it yet.

Reply:
Humm, how would it work since we take the DSP off on suspend. You'd be missing a lot of context and resuming the DMA might not be that useful. What is the expectation here and was it supported in the SKL driver?

Reply:
It is supported in SKL driver. In SOF driver, it is a missing feature at the moment, we are relying on ALSA resume which means the resume point may be not so accurate in my opinion.

Reply:
For my education, where do you see this supported in SKL?